### ISSUE TITLE
Allow adding '--component' flag in the additional parameters for running Cypress Component tests (base version)

### ISSUE THREAD

**Issue by manuelkroiss** [mbolotov/intellij-cypress#105]
Cypress does support component tests (currently in beta)

To run a component test, one has to add the `--component` flag to the `cypress run` command.

However, it seems like the plugin always adds the `--e2e` flag by default.
Thus, adding `--component` as a program argument results in the following error:

```
`--e2e` and `--component` cannot both be passed.

Either pass `--e2e` or `--component`, but not both.
```

Maybe there are further issues related to component tests.

**Comment by mbolotov** [comment]
Hi @manuelkroiss!
Please make sure you have configured your spec to be run in the component mode **in the cypress config file**.
The plugin should automatically add the correct flag to the command line depending on the spec pattern in the config. 
Please also make sure you have the '_automatic_' mode selected in the '_Misc_' tab of the run configuration.

**Comment by manuelkroiss** [comment]
Hi

I did, everything works fine running the tests without the plugin.

Here is my cypress.config.ts
```
export default defineConfig({
  component: {
    devServer: {
      framework: "angular",
      bundler: "webpack",
    },
    specPattern: "**/*.cy.ts"
  }
});
```

Furthermore, I looked for the 'Misc' tab in the run config, but only found a 'General' and a 'Runner' tab. (I am using the newest WebStorm version)

**Comment by mbolotov** [comment]
> I looked for the 'Misc' tab in the run config, but only found a 'General' and a 'Runner' tab

Ohh, that's really strange. 
What version of the plugin do you use?
Can you share the screenshot of your configuration?
This is mine:
![image](https://user-images.githubusercontent.com/19785715/195162478-63fe61d5-f53b-47c3-8361-de9ff01753a3.png)

**Comment by mbolotov** [comment]
Well, I just realized you're probably talking about the base version of plugin, right?
You can try this bundle with the fix:
[Cypress Support-1.5.1-EAP.zip](https://github.com/mbolotov/intellij-cypress/files/9759684/Cypress.Support-1.5.1-EAP.zip)
It will not add `--e2e` parameter if you set `--component` flag in the additional parameters

**Comment by manuelkroiss** [comment]
You were right, I was talking about the base version.

Your EAP version now works as expected. Thanks.

**Comment by mbolotov** [comment]
fixed in 1.5.1 version

**Comment by siffash** [comment]
> Well, I just realized you're probably talking about the base version of plugin, right? You can try this bundle with the fix: [Cypress Support-1.5.1-EAP.zip](https://github.com/mbolotov/intellij-cypress/files/9759684/Cypress.Support-1.5.1-EAP.zip) It will not add `--e2e` parameter if you set `--component` flag in the additional parameters

I have the same issue, and I'm using the latest version 1.5.1. Unfortunately I can't find any additional parameters, this is what I get:
![image](https://user-images.githubusercontent.com/33043735/228815217-7a4d1fdc-0f1a-4c85-8abc-f1de0b76fba8.png)

If I add `--component` as a program argument, it doesn't recognize it and runs again with `--e2e` argument.

**Comment by rdpascua** [comment]
I can't see anything as well on the Webstorm

![image](https://user-images.githubusercontent.com/2498246/229008452-9ac7ba22-e169-4856-a731-637a4ab7c500.png)
